packages/core/card.ts

```diff
@@ -484,10 +537,10 @@ export class Card {
     return data;
   }
 
-  private async serializeIncluded(included: Map<string, OcclusionRulesOrDefaults[]>): Promise<ResourceObject[]> {
+  private async serializeIncluded(included: Map<string, OcclusionRules[]>): Promise<ResourceObject[]> {
```

Comment:
The `serializeIncluded` method is re-serializing the same cards repeatedly, and then spending effort uniquing them back out. I didn't get around to improving that yet.